vnext/Microsoft.ReactNative/ReactRootView.cpp

```diff
@@ -12,7 +12,9 @@ namespace winrt::Microsoft::ReactNative::implementation {
 ReactRootView::ReactRootView() noexcept {
   m_rootControl = std::make_shared<react::uwp::ReactRootControl>(*this);
   UpdatePerspective();
-  Loaded([this](auto &&, auto &&) { react::uwp::SetCompositor(react::uwp::GetCompositor(*this)); });
+  Loaded([this](auto &&, auto &&) {
+    ::Microsoft::ReactNative::SetCompositor(::Microsoft::ReactNative::GetCompositor(*this));
```

Comment:
nit in ReactNativeHost.cpp you added `winrt` namespace explicitly in front.

Reply:
never mind. read XamlView.cpp and that class is not winrt, was confused with double use of namsepaces.

---
In reply to: [508616853](https://github.com/microsoft/react-native-windows/pull/6268#discussion_r508616853) [](ancestors = 508616853)